crates/lib3h/src/engine/real_engine.rs

```diff
@@ -277,6 +286,18 @@ impl<T: Transport, D: Dht> RealEngine<T, D> {
             //   - From GetAuthoringList      : Convert to DhtCommand::BroadcastEntry
             //   - From DHT EntryDataRequested: Convert to DhtCommand::EntryDataResponse
             Lib3hClientProtocol::HandleFetchEntryResult(msg) => {
+                let mut is_data_for_author_list = false;
+                if self.request_track.has(&msg.request_id) {
+                    match self.request_track.remove(&msg.request_id) {
```

Comment:
So the pattern is after we respond to a transaction we throw away the request id, right? And because we don't have persistence, these ids are just cached locally for now. Seems good enough for time being.

Reply:
:+1: hopefully we won't have so many outstanding requests to core that we need them to me anywhere but just in memory... but we can totally evaluate that if it becomes an issue : )